lib/vsc/utils/fancylogger.py

```diff
@@ -186,12 +187,23 @@ def _env_to_boolean(varname, default=False):
 
 
 APOCALYPTIC = 'APOCALYPTIC'
-# register new loglevelname
 logging.addLevelName(logging.CRITICAL * 2 + 1, APOCALYPTIC)
+
 # register QUIET, EXCEPTION and FATAL alias
-logging._levelNames['EXCEPTION'] = logging.ERROR
-logging._levelNames['FATAL'] = logging.CRITICAL
-logging._levelNames['QUIET'] = logging.WARNING
+LOG_LEVEL_ALIASES = {
+    'EXCEPTION': logging.ERROR,
+    'FATAL': logging.CRITICAL,
+    'QUIET': logging.WARNING,
+}
+if sys.version_info[0] < 3:
```

Comment:
only the name of the dict should be part of the if/else, the for loop is identical
```python
if is_py3():
    levelnames = logging._nameToLevel
else:
    levelnames = logging._levelNames

for alias in LOG_LEVEL_ALIASES:
    levelnames[alias] = LOG_LEVEL_ALIASES[alias]
```

Reply:
ok, fixed